reference/7.5/Microsoft.PowerShell.Utility/Format-Table.md

```diff
@@ -141,7 +142,11 @@ Get-Service | Format-Table -Property Name, DependentServices
 displayed in the table.
 
 **Name** and **DependentServices** are two of the object type's properties. To view all the
-properties: `Get-Service | Get-Member -MemberType Properties`.
+properties:
+
+```powershell
+Get-Service | Get-Member -MemberType Properties
+```
```

Comment:
Your change breaks the PlatyPS schema. That is why it was written as a code span rather than a code block.

The structure of an example must be:
- H3 Example X - Title text
- 0 or more paragraphs
- 1 or more code blocks
- 0 or more paragraphs

You can't have more code blocks after the last trailing paragraph.